views/partials/navbar.ejs

```diff
@@ -1,28 +1,56 @@
-<nav class="navbar navbar-expand-lg navbar-light bg-light">
-    <div class="container-fluid">
-      <a class="navbar-brand" href="/">aTom</a>
-      <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
-        <span class="navbar-toggler-icon"></span>
-      </button>
+<nav class="navbar navbar-expand-lg navbar-light bg-light sticky-top">
+  <div class="container-fluid">
+    <a class="navbar-brand" href="#!">
+      <img class="logo" src="/assets/images/logo_pink_1.png" alt="A.tom logo" style="height: 2rem" />
```

Comment:
question: Why inline style?

Reply:
Hm.. sorry.... because that point of time, I wanna do a quick test to see how it is, then ... I leave it there and forget.